web-service/helpers.py

```diff
@@ -132,42 +130,48 @@ def SortByDistanceCmp(a, b):
 
 ################################################################################
 
-def FetchAndStoreUrl(siteInfo, url):
+def GetSiteInfoForUrl(url, rssi=None, txpower=None, force_update=False):
```

Comment:
This is used to recursively look up in cache after each redirection.  We never cache the redirected value, but we always want to cache the final destination.